grunt-configs/shell.js

```diff
@@ -31,6 +31,34 @@ module.exports = function(grunt, options) {
                     cwd: 'facia-tool/public'
                 }
             }
+        },
+        jspmBundleFaciaToolCollections: {
+            command: 'node ../../node_modules/jspm/jspm.js bundle --minify ' + [
```

Comment:
Because of [this](https://github.com/jspm/jspm-cli/issues/564) and in particular [this comment](https://github.com/jspm/jspm-cli/issues/564#issuecomment-76997156).
Later on I `System.import` some other files, so I cannot use `-sfx` because it only support a single point of entrance.